### ISSUE TITLE
Constant visibility false positive on PHP < 7.1

### ISSUE THREAD

**Issue by SoftCreatR** [squizlabs/PHP_CodeSniffer#3154]
In one of my projects, I am using PSR12 as rule. However, there's a branch for older PHP versions (5.4-7.0), which uses the same ruleset:

```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <arg name="basepath" value="."/>
    <arg name="cache" value=".phpcs-cache"/>
    <arg name="colors"/>
    <arg name="extensions" value="php"/>
    <arg name="parallel" value="10"/>

    <arg value="p"/>

    <file>src</file>
    <file>tests</file>

    <rule ref="PSR12"/>
</ruleset>
```

Unfortunately, this fails when running phpcs:

```
WARNING | Visibility must be declared on all constants if your project supports PHP 7.1 or later
```

From the composer.json:

```json
"require": {
    "php": ">=5.4,<7.1"
}
```

Running `phpcbf` on the other hand doesn't report this issue.

Can I disable this single rule in my phpcs.xml?

EDIT:

Answered my own question:

```xml
    <rule ref="PSR12">
        <exclude name="PSR12.Properties.ConstantVisibility"/>
    </rule>
```

**Comment by jrfnl** [comment]
Glad to hear you found the solution. You may want to close the issue now ?

**Comment by SoftCreatR** [comment]
Well, the issue still exists.

**Comment by jrfnl** [comment]
That's inherent to PSR12 demanding it.

Also see: https://github.com/squizlabs/PHP_CodeSniffer/issues/2668#issuecomment-546254280
